src/grt/src/RepairAntennas.cpp

```diff
@@ -272,16 +272,24 @@ void RepairAntennas::addWireTerms(Net* net,
             = layer1->getDirection() == odb::dbTechLayerDir::VERTICAL ? layer1
                                                                       : layer2;
         // create horizontal wire to connect to the pin
-        wire_encoder.newPathVirtualWire(
-            jct_id, h_layer, odb::dbWireType::ROUTED);
+        if (jct_id != -1) {
```

Comment:
Not that important but it might be nice to factor out these 4 lines and the next 4 lines into a small function if that makes sense. The code is virtually identical except for the h_layer/v_layer  difference.

Reply:
It makes sense. I've pushed this refactor.